packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -85,26 +77,54 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
       initialsTextColor="white"
     />
   );
+
+  const meTextStyle = mergeStyles({ color: theme.palette.neutralTertiary }, styles?.me);
+  const contextualMenuStyle = mergeStyles({ background: theme.palette.neutralLighterAlt }, styles?.menu);
+  const menuButtonContainerStyle = mergeStyles({
+    root: {
+      selectors: {
+        '&:hover': { background: 'none' },
+        '&:active': { background: 'none' }
+      }
+    }
+  });
+
   return (
     <div
       ref={containerRef}
-      className={mergeStyles(participantItemContainerStyle(theme), styles?.root)}
-      onClick={showMenu}
+      className={mergeStyles(participantItemContainerStyle, styles?.root)}
+      onMouseEnter={() => setItemHovered(true)}
+      onMouseLeave={() => setItemHovered(false)}
     >
       {avatarToUse}
+      {me && <Stack className={meTextStyle}>{`(you)`}</Stack>}
       {onRenderIcon && (
-        <Stack className={mergeStyles(iconContainerStyle, styles?.iconContainer)}>{onRenderIcon(props)}</Stack>
-      )}
-      {menuItems && (
-        <ContextualMenu
-          items={menuItems}
-          hidden={menuHidden}
-          target={clickEvent ?? containerRef}
-          onItemClick={hideMenu}
-          onDismiss={hideMenu}
-          directionalHint={DirectionalHint.bottomLeftEdge}
-          className={mergeStyles(styles?.menu)}
-        />
+        <Stack horizontal={true} className={mergeStyles(iconContainerStyle, styles?.iconContainer)}>
+          {menuItems && menuItems.length > 0 && (itemHovered || !menuHidden) ? (
+            <div
+              onMouseEnter={() => setMenuButtonHovered(true)}
+              onMouseLeave={() => setMenuButtonHovered(false)}
+              title="More options"
+              className={menuButtonContainerStyle}
+              onClick={() => setMenuHidden(false)}
+            >
+              <MoreIcon outline={!menuButtonHovered} />
+            </div>
+          ) : (
+            onRenderIcon(props)
+          )}
+          {menuItems && (
```

Comment:
nit: should this check the menuItems.length as well here (i.e. don't show the contextual menu if there are no menu items inside the menu)
```suggestion
          {menuItems && menuItems.length > 0 && (
```

Reply:
(I guess this isn't necessary actually as the `menuHidden` will never be false when menuItems.length === 0)